src/BrowserFactory.php

```diff
@@ -16,22 +16,22 @@
 class BrowserFactory
 {
 
-    protected $chromeBinaries;
+    protected $chromeBinary;
 
-    public function __construct(string $chromeBinaries = null)
+    public function __construct(string $chromeBinary = null)
     {
-        // auto guess chrome binaries
-        if (null === $chromeBinaries) {
+        // auto guess chrome binary
+        if (null === $chromeBinary) {
             $envChromePath = getenv('CHROME_PATH');
 
             if ($envChromePath) {
-                $chromeBinaries = '"' . $envChromePath . '"';
+                $chromeBinary = '"' . $envChromePath . '"';
```

Comment:
We don't need to add the quote here then. It breaks the following currently working code.

```php
putenv('CHROME_PATH=C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe');
$browserFactory = new BrowserFactory();
```